### ISSUE TITLE
[Bug] Windows builds in CI fail

### ISSUE THREAD

**Issue by wenzeslaus** [OSGeo/grass#2454]
**Describe the bug**

OSGeo4W build in CI fails with many module builds failing, although many modules are built.

Error messages in the log are unclear to me because I see the same messages for modules which are working.

**To Reproduce**

Go to a recently opened or changed PR and see OSGeo4W check, e.g., #2448 or #2453. View raw logs.

**Screenshots**

```
...
2022-06-20T19:38:17.7334997Z make[4]: Entering directory '/d/a/grass/grass/general/g.copy'
2022-06-20T19:38:17.7710055Z make[4]: Circular ../../mswindows/generic.manifest.rc <- ../../mswindows/generic.manifest.rc dependency dropped.
2022-06-20T19:38:17.7718008Z sed	-e 's/@CMD@/g.copy/' \
2022-06-20T19:38:17.7718472Z 	-e 's/@VER_MAJOR@/8/' \
2022-06-20T19:38:17.7719140Z 	-e 's/@VER_MINOR@/3/' \
2022-06-20T19:38:17.7719827Z 	-e 's/@VER_RELEASE@/dev/' \
2022-06-20T19:38:17.7720399Z 	../../mswindows/generic.manifest.rc > OBJ.x86_64-w64-mingw32/g.copy.manifest.rc
2022-06-20T19:38:17.8191926Z sed 's/@CMD@/g.copy/' ../../mswindows/generic.manifest > OBJ.x86_64-w64-mingw32/g.copy.exe.manifest
2022-06-20T19:38:17.8668138Z windres --input=OBJ.x86_64-w64-mingw32/g.copy.manifest.rc --input-format=rc --output=OBJ.x86_64-w64-mingw32/g.copy.manifest.res --output-format=coff -I../../mswindows
2022-06-20T19:38:17.9853633Z rm OBJ.x86_64-w64-mingw32/g.copy.exe.manifest OBJ.x86_64-w64-mingw32/g.copy.manifest.rc
2022-06-20T19:38:17.9854462Z make[4]: Leaving directory '/d/a/grass/grass/general/g.copy'
2022-06-20T19:38:18.0254098Z C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: warning: --export-dynamic is not supported for PE+ targets, did you mean --export-all-symbols?
2022-06-20T19:38:18.0806825Z if [ "/d/a/grass/grass/dist.x86_64-w64-mingw32/bin/g.copy.exe" != "" ] ; then GISRC=/d/a/grass/grass/dist.x86_64-w64-mingw32/demolocation/.grassrc83 GISBASE=D:/a/grass/grass/dist.x86_64-w64-mingw32 PATH="/d/a/grass/grass/dist.x86_64-w64-mingw32/bin:/d/a/grass/grass/dist.x86_64-w64-mingw32/bin:/d/a/grass/grass/dist.x86_64-w64-mingw32/scripts:$PATH" PYTHONPATH="D:/a/grass/grass/dist.x86_64-w64-mingw32/etc/python;D:/a/grass/grass/dist.x86_64-w64-mingw32/gui/wxpython;$PYTHONPATH" PATH="/d/a/grass/grass/dist.x86_64-w64-mingw32/bin:/d/a/grass/grass/dist.x86_64-w64-mingw32/bin:/d/a/grass/grass/dist.x86_64-w64-mingw32/scripts:/d/a/grass/grass/dist.x86_64-w64-mingw32/lib:/d/a/grass/grass/dist.x86_64-w64-mingw32/lib:/c/OSGeo4W/bin:/usr/bin:/mingw64/bin" LC_ALL=C LANG=C LANGUAGE=C /d/a/grass/grass/dist.x86_64-w64-mingw32/bin/g.copy.exe --html-description < /dev/null | grep -v '</body>\|</html>' > g.copy.tmp.html ; fi
2022-06-20T19:38:18.1458801Z make[3]: *** [../../include/Make/Html.make:14: g.copy.tmp.html] Error 1
2022-06-20T19:38:18.1459405Z rm g.copy.tmp.html
2022-06-20T19:38:18.1460269Z make[3]: Leaving directory '/d/a/grass/grass/general/g.copy'
...
2022-06-20T19:48:44.5432551Z Errors in:
2022-06-20T19:48:44.5432899Z /d/a/grass/grass/lib/raster3d/test
2022-06-20T19:48:44.5433269Z /d/a/grass/grass/display/d.barscale
2022-06-20T19:48:44.5433632Z /d/a/grass/grass/display/d.colortable
2022-06-20T19:48:44.5433986Z /d/a/grass/grass/display/d.erase
...
2022-06-20T19:48:44.5527226Z /d/a/grass/grass/imagery/i.pca
2022-06-20T19:48:44.5527582Z /d/a/grass/grass/imagery/i.vi
2022-06-20T19:48:44.5527933Z /d/a/grass/grass/imagery/i.fft
2022-06-20T19:48:44.5528271Z /d/a/grass/grass/imagery/i.ifft
2022-06-20T19:48:44.5528620Z /d/a/grass/grass/imagery/i.zc
2022-06-20T19:48:44.5528973Z /d/a/grass/grass/ps/ps.map
2022-06-20T19:48:44.5529266Z --
```

**System description**

- Operating System: Windows
- GRASS GIS version: main
- Environment: GitHub Actions VM

**Comment by wenzeslaus** [comment]
No idea about the `< /dev/null` part, I can only say it is not `> /dev/null`.

But instead of lifting up restrictions for the main branch, you can simply try in a PR.

**Comment by ninsbl** [comment]
Fixed in #2508 and backported in #2510
